core/src/test/java/io/grpc/internal/AutoConfiguredLoadBalancerFactoryTest.java

```diff
@@ -884,13 +902,18 @@ public void updateAddresses(List<EquivalentAddressGroup> addrs) {
     }
   }
 
-  private static final class FakeLoadBalancerProvider extends LoadBalancerProvider {
+  private static class FakeLoadBalancerProvider extends LoadBalancerProvider {
```

Comment:
Why remove `final`?

Reply:
it is used in mock to verify and to capture the argument in `handleResolvedAddressGroups_propagateLbConfigToDelegate()`.